scylla/datadog_checks/scylla/data/conf.yaml.example

```diff
@@ -0,0 +1,149 @@
+    #   - scylla.memory
+    #   - scylla.memtables
+    #   - scylla.query
+    #   - scylla.scheduler
+    #   - scylla.sstables
+    #   - scylla.thrift
+    #   - scylla.tracing
+
+    ## @param tags - list of key:value elements - optional
+    ## List of tags to attach to every metrics, events and service checks emitted by this integration.
+    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    #
+    # tags:
+    #   - <KEY_1>:<VALUE_1>
+    #   - <KEY_2>:<VALUE_2>
+
+    ## @param timeout - integer - optional - default: 10
+    ## Overrides the default timeout value in second
+    #
+    # timeout: 10
+
+    ## @param health_service_check - boolean - optional - default: true
+    ## Send a service check reporting about the health of the prometheus endpoint
+    ## It's named scylla.prometheus.health
+    #
+    # health_service_check: true
+
+    ## @param label_to_hostname - string - optional
+    ## Override the hostname with the value of one label.
+    #
+    # label_to_hostname: <LABEL>
+
+    ## @param label_joins - object - optional
+    ## The label join allows to target a metric and retrieve it's label via a 1:1 mapping
+    #
+    # label_joins:
+    #   target_metric:
+    #     label_to_match: <MATCHED_LABEL>
+    #     labels_to_get:
+    #       - <EXTRA_LABEL_1>
+    #       - <EXTRA_LABEL_2>
+
+    ## @param labels_mapper - list of key:value elements - optional
+    ## The label mapper allows you to rename labels.
+    ## Format is <LABEL_TO_RENAME>: <NEW_LABEL_NAME>
+    #
+    # labels_mapper:
+    #   flavor: origin
+
+    ## @param type_overrides - list of key:value elements - optional
+    ## Type override allows you to override a type in the prometheus payload
+    ## or type an untyped metrics (they're ignored by default)
+    ## Supported <METRIC_TYPE> are `gauge`, `counter`, `histogram`, `summary`
+    #
+    # type_overrides:
+    #   <METRIC_NAME>: <METRIC_TYPE>
+
+    ## @param send_histograms_buckets - boolean - optional - default: true
+    ## Set send_histograms_buckets to true to send the histograms bucket.
+    #
+    # send_histograms_buckets: true
+
+    ## @param send_distribution_buckets - boolean - optional - default: false
+    ## Set `send_distribution_buckets` to `true` to send histograms as Datadog distribution metrics.
+    ##
+    ## Learn more about distribution metrics: https://docs.datadoghq.com/developers/metrics/distributions/
+    #
+    # send_distribution_buckets: false
+
+    ## @param send_monotonic_counter - boolean - optional - default: true
+    ## Set send_monotonic_counter to true to send counters as monotonic counter.
+    #
+    # send_monotonic_counter: true
+
+    ## @param send_distribution_counts_as_monotonic - boolean - optional - default: false
+    ## Set send_distribution_counts_as_monotonic to true to send histograms & summary
+    ## counters as monotonic counters (instead of gauges).
+    #
+    # send_distribution_counts_as_monotonic: false
```

Comment:
Should these lines be in the config?  Don't think we really want users to change from the defaults here, right?

Reply:
These options are available, but it doesn't seem like we document these in the other prometheus integrations

Reply:
I'm leaning towards setting `send_histograms_buckets` to `true` explicitly in the code, and removing these options from the conf example.  I have a small writeup that describes my investigation of these options that I'll add to the wiki.

Reply:
Removed them :)